atomic

```diff
@@ -513,6 +514,16 @@ def create_parser(help_text):
                      help=_("Only display container IDs"))
     add_opt(pss)
 
+    # atomic sign
+    signp = subparser.add_parser("sign", help="Sign an image",
+                                 epilog="Create a signature for an image which can be used later to verify it.")
+    signp.set_defaults(_class=Sign, func="sign")
+    signp.add_argument("sign_targets", nargs="*", help=_("images to sign"))
+    signp.add_argument("--sign_key", dest="sign_key", default=None,
+                       help=_("Name of the signing key"))
```

Comment:
Maybe mention that the default can be set in /etc/atomic.conf or wherever it is set.